extensions/sns/src/main.rs

```diff
@@ -38,42 +34,67 @@ pub struct SnsOpts {
 /// Subcommands of `dfx sns`
 #[derive(Parser)]
 enum SubCommand {
-    /// Subcommands for working with configuration.
+    /// Deploy an sns directly to a subnet, skipping the sns-wasms canister.
+    /// The SNS canisters remain controlled by the developer after deployment.
```

Comment:
fyi

This seems like a pretty good explanation of what a testflight is.

Reply:
Thanks, I used it in the readme!